### PR TITLE
Column View Improvements #2

### PR DESCRIPTION
**Pull request by TyJOrtiz**
**Resolved / Related Issues**
Itemize resolved / related issues by this PR.
- Related #4397 

**Details of Changes**
Add details of changes here.
- Fixed issue where unfocused listview item color wouldnt change

**Validation**
How did you test these changes?
- [X] Built and ran the app
- [ ] Tested the changes for accessibility

**Screenshots (optional)**
Add screenshots here.
image

but no chevron

### CODE REVIEW

File: Files/Views/LayoutModes/ColumnViewBase.xaml [hunk 1]
```diff
@@ -56,60 +1065,61 @@
...
+                                </Image.Source>
+                            </Image>
+                            <Viewbox
                         x:Name="TypeUnknownGlyph"
                         MaxWidth="24"
                         MaxHeight="24"
                         HorizontalAlignment="Stretch"
                         VerticalAlignment="Stretch"
-                        x:Load="{x:Bind LoadUnknownTypeGlyph, Mode=OneWay}"
+                        Visibility="{x:Bind LoadUnknownTypeGlyph, Mode=OneWay}"
```
Comment (yaichenbaum) [inline Files/Views/LayoutModes/ColumnViewBase.xaml:1118]: Can you continue using x:Load?

**Comment by TyJOrtiz** [comment]
x:Load is causing issues with the x:bind, it’s related to the issue where xaml can’t find anything in a ContentTemplateRoot (might be WinUI related as it was working before)

Sent from my iPhone

On Apr 1, 2021, at 5:25 PM, Yair Aichenbaum ***@***.***> wrote:

﻿

@yaichenbaum commented on this pull request.

________________________________

In Files/Views/LayoutModes/ColumnViewBase.xaml<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Ffiles-community%2FFiles%2Fpull%2F4418%23discussion_r606013203&data=04%7C01%7C%7Cd06b94e00af64222775708d8f56dd7dc%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637529199400305802%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=PCbNNTLps2Qhbd%2FNLJAX%2FfGP%2BAK02jfiIv2CzFGlVjY%3D&reserved=0>:

>                          x:Name="TypeUnknownGlyph"
                         MaxWidth="24"
                         MaxHeight="24"
                         HorizontalAlignment="Stretch"
                         VerticalAlignment="Stretch"
-                        x:Load="{x:Bind LoadUnknownTypeGlyph, Mode=OneWay}"
+                        Visibility="{x:Bind LoadUnknownTypeGlyph, Mode=OneWay}"


Can you continue using x:Load?

File: Files/Views/LayoutModes/ColumnViewBase.xaml (hunk 1 above)
Comment (d2dyno1) [reply Files/Views/LayoutModes/ColumnViewBase.xaml:1118]: Could you change the name to UnknownTypeGlyphVisibility then @TyJOrtiz

Comment (TyJOrtiz) [reply Files/Views/LayoutModes/ColumnViewBase.xaml:1118]: There is no `UnknownTypeGlyphVisibility ` in ListedItem

Comment (d2dyno1) [reply Files/Views/LayoutModes/ColumnViewBase.xaml:1118]: Is LoadUnknownTypeGlyph a boolean? Then it won't work with Visibility property

Comment (TyJOrtiz) [reply Files/Views/LayoutModes/ColumnViewBase.xaml:1118]: Windows 10 has supported BoolToVisibiltiyConverter natively since 15063

**Pull request merged**